dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -743,6 +746,10 @@ private Response getFileByToolVersionID(String registryId, String versionIdParam
                         dockerfile.setOriginalFile(potentialDockerfile.get());
                         dockerfile.setImageType(new EmptyImageType());
                         toolVersion.setContainerfile(true);
+                        dockerfile.setDockstoreAbsolutePath(MoreObjects.firstNonNull(potentialDockerfile.get().getAbsolutePath(), ""));
+                        String selfPath = value.getUriInfo().getRequestUri().toASCIIString();
```

Comment:
I'm not sure this will work 100% correctly in a deployed environment.

We log from [here](https://github.com/dockstore/dockstore/blob/3426310cedb4918e7bf227d2c6c676a2ccc69887/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/DiagnosticsHelper.java#L213); in the CloudWatch logs, I see the logged value in staging `http://staging.dockstore.org...` (note `http` instead of `https`). I'm guessing because the LB has already terminated the https connection.

Our http urls will get redirected to https urls, but ideally they should be be https to begin with.